draft-ietf-quic-transport.md

```diff
@@ -682,11 +682,11 @@ sequence of frames, as described in {{frames}}.
 
 ## Connection ID {#connection-id}
 
-QUIC connections are identified by their 64-bit Connection ID.  All long headers
-contain a Connection ID.  Short headers indicate the presence of a Connection ID
-using the Omit Connection ID flag.  When present, the Connection ID is in the
-same location in all packet headers, making it straightforward for middleboxes,
-such as load balancers, to locate and use it.
+QUIC connections are identified by their 136-bit Connection ID.  All long
```

Comment:
You lower-cased all occurrences of "Connection ID" in this paragraph except this one.